CHANGELOG.md

```diff
@@ -103,6 +103,13 @@ For more details see: [Deprecated in 4.0]
 [API Changes from 3.0 to 4.0]: https://docs.fluidproject.org/infusion/development/apichangesfrom3_0to4_0
 [Deprecated in 4.0]: https://docs.fluidproject.org/infusion/development/deprecatedin4_0
 
+## [3.0.1] - 2022-05-10
```

Comment:
This is actually a reference-style link which needs to be defined at the bottom of the file. This keeps things more readable but also allows for re-use of the version link.

Reply:
Thanks, I didn't know about these